### ISSUE TITLE
Examples with Set-Content using "-Encoding Byte" don't work  but "-AsByteStream" works

### ISSUE THREAD

**Issue by funkypenguin** [MicrosoftDocs/office-docs-powershell#4171]
Hey guys,

Powershell n00b here, but the examples above didn't work for me. Here's an example:

```
Set-Content : Cannot process argument transformation on parameter 'Encoding'. 'Bytes' is not a supported encoding name. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method.
Parameter name: name
At line:1 char:60
+ Set-Content -path "tmp/Rules.xml" -Value "hello" -Encoding Bytes
+                                                            ~~~~~
+ CategoryInfo          : InvalidData: (:) [Set-Content], ParameterBindingArgumentTransformationException
+ FullyQualifiedErrorId : ParameterArgumentTransformationError,Microsoft.PowerShell.Commands.SetContentCommand
```

However, after digging a bit on how Set-Content works, I took a shot in the dark and tried `-AsByteStream` instead of `-Encoding Byte`, which worked :)

So, either I'm a dumbass who doesn't understand Powershell properly, or these docs need an update ;)

D

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f2dae54d-bc59-bf36-8542-cd952250ad31
* Version Independent ID: 38ebb939-f898-a594-5d44-61b0c0703294
* Content: [Export-TransportRuleCollection (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/policy-and-compliance/Export-TransportRuleCollection?redirectedfrom=MSDN&view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/policy-and-compliance/Export-TransportRuleCollection.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/policy-and-compliance/Export-TransportRuleCollection.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by chrisda** [comment]
Apparently, you're using PowerShell 6.0. The Byte value is available on the Encoding parameter in PowerShell 3.0 to PowerShell 5.1, but not in PowerShell 6.0 (compare the Encoding parameter description at the links below):

https://docs.microsoft.com/powershell/module/microsoft.powershell.management/set-content?view=powershell-5.1

https://docs.microsoft.com/powershell/module/microsoft.powershell.management/set-content?view=powershell-6

I added a note to the first example to replace `-Encoding Byte` with `-AsByteSteam` in PowerShell 6.0 or later. Also note that this same issue affects the Import-TransportRuleCollection cmdlet, so I added the same note there.

Thanks for the feedback.
